measurement-kit-test/Base.lproj/LaunchScreen.storyboard

```diff
@@ -1,7 +1,8 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I actually don't know what this is, it's a change introduced by the `interface builder`